### ISSUE TITLE
Switch to EXCALIDRAW VIEW in the MORE OPTIONS menu of this document.

### ISSUE THREAD

**Issue by airtonix** [MSzturc/obsidian-advanced-slides#113]
![image](https://user-images.githubusercontent.com/61225/182092883-3d8a4acf-75a0-448d-9937-934f7c27173a.png)

Not sure what I'm supposed to do?

**Comment by TimothyGutierrez** [comment]
Likewise, having issues opening excalidraw image in advanced slides.
code matches https://mszturc.github.io/obsidian-advanced-slides/extend-syntax/excalidraw/

Output is as attached image.
<img width="612" alt="Screen Shot 2022-08-01 at 19 47 42" src="https://user-images.githubusercontent.com/90198967/182132515-dc71f1ab-506d-488c-9b7c-a56051e519ae.png">

Obsidian is V0.15.9
Advanced Slides Version 1.15.0

**Comment by airtonix** [comment]
@TimothyGutierrez sorry you're issue is different. this is about the preview within Obsidian.

How did you get url? where is this documented

**Comment by TimothyGutierrez** [comment]
> @TimothyGutierrez sorry you're issue is different. this is about the preview within Obsidian.
> 
> How did you get url? where is this documented

The preview within Obsidian in Advanced Slides is the "open in browser" button on top of slideshow preview. The button looks like a monitor.
As for where this is documented, that is a good question. I cannot find it now. Perhaps I saw it in a video demonstration or I just pressed the button.

**Comment by airtonix** [comment]
yep, looks like the theme i had enabled was hiding most of the controls. 

switched to Minimal and "things" are more obvious now.

Still don't know why I'm getting the yellow band of death ☝🏻

**Comment by TimothyGutierrez** [comment]
> @TimothyGutierrez sorry you're issue is different. this is about the preview within Obsidian.

You are correct @airtonix. Apologies for hijacking your thread. I have opened a different thread to address my issue.

**Comment by MSzturc** [comment]
@airtonix: Please have a look at the documentation: https://mszturc.github.io/obsidian-advanced-slides/extend-syntax/excalidraw/

**Comment by airtonix** [comment]
@MSzturc Might I suggest that the `yellow banner of death` above say exactly that?

I can imagine it would perhaps result in:

- more users not removing the plugin and you never hearing from them.
- Me (or people like me) never creating a issue ticket like this again.

just some thoughts.

**Comment by MSzturc** [comment]
There is no good way to solve this problem since the yellow banner is part of an excalidraw markdown itself.

**Comment by simonmeggle** [comment]
I also had this error message. 
In my case, the solution was to simply add ".excalidraw" as extension to the diagram note. 
As soon as I remove it, the error comes again. 
🤷

**Comment by sdzialowski-godaddy** [comment]
Simply install excalidraw plugin. https://youtu.be/QKnQgSjJVuc?si=tAGoIgunp9XjyFkt&t=425

This message 

```
==⚠  Switch to EXCALIDRAW VIEW in the MORE OPTIONS menu of this document. ⚠==
```
 usually happens when one opens existing vault for the first time.

Seems like community plugins are installed per vault, or something ??? 🤷‍♂️ Dunno. 
Installing the plugin seems to help and it makes sense to me that obviously it will not work in the plugin absence.

**Comment by simonmeggle** [comment]
@sd-godaddy That does not make sense to me at all. 
Users getting this error have already installed the excalidraw plugin and they _expect_ the drawing to appear. (At least me)

> usually happens when one opens existing vault for the first time.

I open my vault every day.

> Installing the plugin seems to help

So, you really had the error and could solve it just by installing the plugin?

**Comment by BlueClaypot** [comment]
I found a way to fix the issue! 

1. Create a new Excalidraw drawing
2. Open more options ( ⋮ ), then click Open as Markdown.
4. Go back to the drawing where you're getting the _Switch to Excalidraw view_ error.
5. Copy all of the markdown text except for the warning message.
6. Paste the markdown text into the new Excalidraw document.
7. Open more options, then click Open as Excalidraw Drawing.

Hope that helps!

**Comment by lightbluepoppy** [comment]
I could not find the option _Open as Excalidraw Drawing_ as I tried to add tags to the tags property in the frontmatter. After I added tags using `#` below the message `==⚠  Switch to EXCALIDRAW VIEW in the MORE OPTIONS menu of this document  ⚠==`  OR adding tags BEFORE `[excalidraw]` tag, I could find the _Open as Excalidraw Drawing_ option at the very top of More Options ⋮.

**Comment by mahemoff** [comment]
@sdzialowski-godaddy "Seems like community plugins are installed per vault, or something ??? 🤷‍♂️ Dunno."

Yes, plugin setup is vault-specific, but the reason this could happen even in a single vault is that plugin setup isn't necessarily synced between Obsidian instances. I just encountered the issue because a drawing from my phone wasn't showing on PC. As you noted, installing and enabling Excalidraw fixed it.

**Comment by stroiman** [comment]
TLDR; Be sure the excalidraw file has `.excalidraw.md` file extension.

I just had this issue, and excalidraw by default creates `.md` files, not `.excalidraw.md`. By renaming, adding the `.excalidraw` part, advanced slides worked as intentional.

The default excalidraw file extension `.md` or `.excalidraw.md` can be configured in excalidraw's settings. I think the default behaviour might have changed for this.

> [!NOTE]
> When renaming, you don't set the `.md` part of the filename yourself. Obsidian does that for you; just add `.excalidraw` to the file that doesn't work.

**Comment by ikesler** [comment]
I've just got this problem and fixed it by adding `excalidraw-plugin: parsed` to the frontmatter. I think, I accidentally removed it by a script
